### DISCUSSION TITLE
Confused about the target platforms for Nuget package

### DISCUSSION THREAD

**Discussion by serefarikan** [microsoft/FASTER#567] [Q&A] [answered]
Hi, 
First of all, it's great to have a discussion forum!. Makes a huge difference compared to having to open issues to ask questions. 

Looking at the nuget package explorer for the core Faster nuget package:
https://nuget.info/packages/Microsoft.FASTER.Core/1.9.6

I am a bit confused. Why is there .NET Standard 2.0 and 2.1 both? Given 2.1 should be backward compatible, why is there an explicit 2.1? 2.0 should be consumable by .NET Standard 2.1 projects. 

There is also net461, so I'm under the impression that all functionality is supported for these TFMs. Would not this make 2.1 redundant?

**Comment by badrishc** [comment, marked as answer]
We have .NET standard 2.0 compatibility so that UWP apps can use FASTER. There are some more efficient APIs available in 2.1 that we conditionally compile into the binary for 2.1, so if you use that one you get the performance benefits.
